### DISCUSSION TITLE
Solax based on mqtt/additional Sensor ?

### DISCUSSION THREAD

**Discussion by Phil2204** [wills106/homeassistant-solax-modbus#517] [General]
@wills106 it is working Fine with your Solution on home Assistant. Wiring is with waveshare from @alienatedsec (thanks to you) . I am wondering if solax modbus is based on mqtt? I would like to use the data for evcc (addon in Home Assistant) in Order to charge my car with PV. 

Another question is if I can das further sensors (e.g grid Import/Export per day or calculated values like autarky) 

Thanks for your Support 

Best, Philipp

**Comment by wills106** [comment]
This integration doesn't use mqtt. It's direct Modbus between the Inverter and Home Assistant.
Does it need to be mqtt or can you just point it to a sensor responding to what you are exporting?

Import / Export per day should already be present. The total counts are disabled by default though.

**Reply by Phil2204** [reply]
there are several possibilities to get the data. see here: https://docs.evcc.io/en/docs/reference/plugins

as I understand is, that modbus connection is not for more than 1 device possible. so therefore, I thought to use mqtt. Do you have another idea?

**Reply by alienatedsec** [reply]
You can always advertise your inverter data to MQTT CIA Home Assistant. I am not sure what is required for the EVCC to work coreectly as i haven't studied it documentation yet.

**Reply by wills106** [reply]
You could also look at using a webhook to connect the data from HA to EVCC?

**Reply by Phil2204** [reply]
sorry for my late respond. I am not sure about what is webhook and if this is working with EVCC. MQTT is working with this. @alienatedsec can you give me further introduction for configuring mqtt ? I am able to fill in the "MQTT sensors" in the config of EVCC
